README.md

```diff
@@ -1,18 +1,680 @@
+
+Make sure you have [Yarn installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable).
+
+In the root of the repo run `yarn`. This installs the Docusaurus framework packages and the other packages that help generate the `build` output.
+
+Change directory into `assembly` and run `yarn` there as well. This installs the Assembly Workflow dependencies that generates information nodes and generates the guides into `/docs` .
+
+### How to follow style guidance
+
+In general, Temporal content follows the [Google developer documentation style guide](https://developers.google.com/style). When the Google guide is silent about an issue, we follow the [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/). And we also maintain a set of Temporal-specific style guidelines that override the Google and Microsoft guides defined below.
+
+But we recommend that you use the Vale extension for your IDE and follow the suggestions.
+
+If you have VSCode, we recommend the Vale VSCode extension.
+
+For details on the styles, see the rulesets defined in the `vale/styles` configuration.
+
+#### Capitalization of core terms
+
+Many of Temporal's core terms can be used in a generic way.
+
+To differentiate one of Temporal's core terms from a generic instance of a term, always treat the Temporal term as a proper noun in documentation.
+
+Generic versions of the same term should not be capitalized and should be used sparingly to avoid confusion.
+
+- Correct: "Next, register the Activity within the Workflow."
+
+- Incorrect: "Next, register the activity within the workflow."
+
+#### Abbreviation of "identifier"
+
+In text, do not abbreviate the word "identifier" as "ID", "Id", or "id" unless it is part of a Temporal core term, such as "Workflow Id" or "Activity Id".
+
+- Correct: "You can provide an order identifier or customer identifier as a Workflow Id."
+
+- Incorrect: "You can provide an order ID or customer id as a Workflow Id."
+
+In code (and when quoting or referring to code in text), follow the conventions of each language.
+
+#### En dashes in ranges
+
+Using an en dash (`&ndash;` or the character `–`) in a range of numbers is acceptable.
+Even better is to use words such as _from_, _to_, and _through_.
+
+#### Be consistent.
+
+If you use an en dash in one range, use en dashes in all ranges.
+Do not mix words and en dashes (or hyphens, for that matter).
+
+- Correct: "5 to 10 GB"
+- Correct: "5–10 GB"
+- Correct: "5-10 GB"
+- Incorrect: "from 5-10 GB"
+
+#### Infinitive verb forms in headings
+
+Titles and headings should use infinitive verb forms whenever possible. People tend to search by using infinitive verb forms, so using them helps SEO.
+
+- Correct: "How to install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Infinitive verb forms in labels
+
```

Comment:
```suggestion
Treat labels like headings or titles and use infinitive verb forms when possible.
```

Reply:
Actually titles and headers are questions & labels are infinitive -
This section will need a rewrite soon.
@djmagee can do that on his pass with style guide changes.